include/tvm/runtime/packed_func.h

```diff
@@ -117,16 +170,12 @@ class PackedFunc {
    * \param rv The return value.
    */
   inline void CallPacked(TVMArgs args, TVMRetValue* rv) const;
-  /*! \return the internal body function */
-  inline FType body() const;
   /*! \return Whether the packed function is nullptr */
-  bool operator==(std::nullptr_t null) const { return body_ == nullptr; }
+  bool operator==(std::nullptr_t null) const { return data_ == nullptr; }
   /*! \return Whether the packed function is not nullptr */
-  bool operator!=(std::nullptr_t null) const { return body_ != nullptr; }
+  bool operator!=(std::nullptr_t null) const { return data_ != nullptr; }
```

Comment:
Let's consider more carefully about the definition of nullability of PackedFunc. There are two cases:

- C1. `PackedFunc::data_` is null
- C2. `PackedFunc::data_::f_call_packed_` is null

How do we formally define if a PackedFunc is null then?

Reply:
Actually I think we can prove that `PackedFunc::data_::f_call_packed_` is never null, because it's only constructible via `make_object<PackedFuncSubObj>` by the constructor of `PackedFunc`. Therefore, I think it might be a good idea to remove these two comparator methods :-)